opencga-server/src/main/java/org/opencb/opencga/server/generator/config/RestMethodParse.java

```diff
@@ -0,0 +1,42 @@
+package org.opencb.opencga.server.generator.config;
+
+public class RestMethodParse {
+
+    private String rest;
+    private String methodName;
```

Comment:
Can't this "methodName" be part of org.opencb.opencga.server.generator.config.Command ? It already has a "rename" ...